_data/create.yml

```diff
@@ -0,0 +1,204 @@
+
+- name: "autoStartRecording"
+  required: false
+  type: "Boolean"
+  description: "Whether to automatically start recording when first user joins (default <code class=\"language-plaintext highlighter-rouge\">false</code>).<br><br> When this parameter is <code class=\"language-plaintext highlighter-rouge\">true</code>, the recording UI in BigBlueButton will be initially active. Moderators in the session can still pause and restart recording using the UI control.&lt;br/<br> NOTE: Don’t pass <code class=\"language-plaintext highlighter-rouge\">autoStartRecording=false</code> and <code class=\"language-plaintext highlighter-rouge\">allowStartStopRecording=false</code> - the moderator won’t be able to start recording!"
+
+- name: "allowStartStopRecording"
+  required: false
+  type: "Boolean"
+  description: "Allow the user to start/stop recording. (default true)<br><br> If you set both <code class=\"language-plaintext highlighter-rouge\">allowStartStopRecording=false</code> and <code class=\"language-plaintext highlighter-rouge\">autoStartRecording=true</code>, then the entire length of the session will be recorded, and the moderators in the session will not be able to pause/resume the recording."
+
+- name: "webcamsOnlyForModerator"
+  required: false
+  type: "Boolean"
+  description: "Setting <code class=\"language-plaintext highlighter-rouge\">webcamsOnlyForModerator=true</code> will cause all webcams shared by viewers during this meeting to only appear for moderators (added 1.1)"
+
+- name: "logo"
+  required: false
+  type: "String"
+  description: "Setting <code class=\"language-plaintext highlighter-rouge\">logo=http://www.example.com/my-custom-logo.png</code> will replace the default logo in the Flash client. (added 2.0)"
+
+- name: "bannerText"
+  required: false
+  type: "String"
+  description: "Will set the banner text in the client. (added 2.0)"
+
+- name: "bannerColor"
+  required: false
+  type: "String"
+  description: "Will set the banner background color in the client. The required format is color hex #FFFFFF. (added 2.0)"
+
+- name: "copyright"
+  required: false
+  type: "String"
+  description: "Setting <code class=\"language-plaintext highlighter-rouge\">copyright=My custom copyright</code> will replace the default copyright on the footer of the Flash client. (added 2.0)"
+
+- name: "muteOnStart"
+  required: false
+  type: "Boolean"
+  description: "Setting <code class=\"language-plaintext highlighter-rouge\">true</code> will mute all users when the meeting starts. (added 2.0)"
+
+- name: "allowModsToUnmuteUsers"
+  required: false
+  type: "Boolean"
+  default: "false"
+  description: "Setting to <code class=\"language-plaintext highlighter-rouge\">true</code> will allow moderators to unmute other users in the meeting. (added 2.2)"
+
+- name: "lockSettingsDisableCam"
+  required: false
+  type: "Boolean"
+  default: "false"
```

Comment:
I think we should use `false` rather than `"false"` when giving default value of a Boolean. Here and the rest of the pull request. What do you say?

Reply:
I don't see the advantage, but you are free to change that pr to anything you like. Just one note: If you use a boolean here, you have to update the table template, otherwise `false` defaults are not shown.